test/e2e/test_cluster_configs/no_outbound.json

```diff
@@ -1,5 +1,8 @@
 {
 	"env": {},
+	"options": {
```

Comment:
@mboersma FYI we need to exclude 1.18 from this test until our VHDs have 1.18 support